keyboards/gray_studio/space65/readme.md

```diff
@@ -1,11 +1,13 @@
 # Gray Studio Space65
 
-A 65% keyboard with RGB underglow, backlighting and USB C, whose design was inspired by the Voyager I space probe and Apple II home computer.
+A 65% keyboard with RGB underglow, backlighting, and USB C. Its design was inspired by the Voyager I space probe and Apple II home computer.
 
 Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)  
 Hardware Supported: Gray Studio Space65  
 Hardware Availability: [Geekhack Group Buy](https://geekhack.org/index.php?topic=98768.0)
 
+**Reset Sequence:** Using this firmware sets `BOOTLOADER_ENABLE` to `lite`. While plugging in, hold the top left key, commonly programmed as `Esc` to put your board into bootloader mode. 
```

Comment:
```suggestion
**Reset Sequence:** `BOOTMAGIC_ENABLE` is set to [`lite`](/docs/feature_bootmagic.md#bootmagic-lite) on this board. Hold the top left key (commonly programmed as Esc) while plugging in to put your board into bootloader mode.
```
@mechmerlin Better?

Reply:
I still disagree because what if you have kbdfans firmware? It's going to be space+b there.